File: Topic9_Stack/InBuiltStack.py
if __name__ == "__main__":
    numstack = []

    numstack.append(4)
    numstack.append(8)
    numstack.append(15)
    numstack.append(16)

   
    print("Stack after pushes:", numstack)

   
    popped_element = numstack.pop()
    print(f"Popped: {popped_element}")


    if numstack: 
        print(f"Peek: {numstack[-1]}")
    else:
        print("Stack is empty, no element to peek")

    print("Stack after popping:", numstack)

File: Topic9_Stack/InbuiltStack.java
package Topic9_Stack;

import java.util.Stack;

public class InbuiltStack {

    public static void main(String[] args) {
        
        Stack<Integer> numStack=new Stack<>();

        numStack.push(1);
        numStack.push(2);
        numStack.pop();
        numStack.push(3);
        numStack.push(4);

        System.out.println(numStack);

    }
    
}
